TypeScript.ContractGenerator/Internals/TypeScriptImportFromUnitStatement.cs

```diff
@@ -6,11 +6,13 @@ internal class TypeScriptImportFromUnitStatement : TypeScriptImportStatement
     {
         public override string GenerateCode(ICodeGenerationContext context)
         {
-            return $"import {{ {TypeName} }} from '{context.GetReferenceFromUnitToAnother(CurrentUnit.Path, TargetUnit.Path)}';";
+            var typeKeyword = UseTypeKeyword ? "type " : "";
+            return $"import {typeKeyword}{{ {TypeName} }} from '{context.GetReferenceFromUnitToAnother(CurrentUnit.Path, TargetUnit.Path)}';";
         }
 
         public string TypeName { get; set; }
         public TypeScriptUnit TargetUnit { get; set; }
         public TypeScriptUnit CurrentUnit { get; set; }
+        public bool UseTypeKeyword { get; set; }
     }
-}
```

Comment:
we also want codestyle tests to be green [here](https://github.com/skbkontur/TypeScript.ContractGenerator/actions/runs/6770752601/job/18485244783)